cli/src/test/java/keywhiz/cli/commands/AddActionTest.java

```diff
@@ -87,22 +90,27 @@ public void addCallsAddForSecret() throws Exception {
     when(keywhizClient.getSanitizedSecretByName(secret.getName()))
         .thenThrow(new NotFoundException()); // Call checks for existence.
 
-    when(keywhizClient.createSecret(secret.getName(), "", content, secret.getMetadata(), 1136214245))
+    when(
+        keywhizClient.createSecret(secret.getName(), "", content, secret.getMetadata(), 1136214245))
         .thenReturn(secretDetailResponse);
 
     addAction.run();
-    verify(keywhizClient, times(1)).createSecret(secret.getName(), "", content, secret.getMetadata(), 1136214245);
+    verify(keywhizClient, times(1)).createSecret(secret.getName(), "", content,
+        secret.getMetadata(), 1136214245);
   }
 
   @Test
   public void addCallsAddForClient() throws Exception {
     addActionConfig.addType = Arrays.asList("client");
     addActionConfig.name = client.getName();
+    addActionConfig.description = "description";
+    addActionConfig.spiffeUri = "spiffe//test.com/client";
```

Comment:
minor nit, but `example.org` is the best choice for test data, as it is explicitly set aside for the purposes of testing -- not that it matters too much in cases like this